src/views/gallery.coffee

```diff
@@ -88,10 +88,8 @@ class GalleryView
 
   layout: =>
     # (1) Find appropriate number of rows by dividing the sum of ideal photo widths by the width of the viewport
-    $(document.body).css('overflowY', 'scroll')
```

Comment:
I have tried removing these two calls locally in `press` module and the gallery still works as expected, so it should fix the problem described in the issue.

Reply:
Why does it set it in the first place?

Reply:
> Why does it set it in the first place?

`overflow-y: scroll` ensures a vertical scrollbar is displayed. Unless you're on macOS, the width of this scrollbar will affect the `viewport_width` calculation below. 

> I have tried removing these two calls locally in press module and the gallery still works as expected, so it should fix the problem described in the issue.

Did you try it with windows/linux on browserstack?

Reply:
Good point @kamilmielnik. I have enabled scrollbars to be always visible in Settings:

![Screen Shot 2019-11-20 at 12 53 31](https://user-images.githubusercontent.com/4209081/69236937-dd0e7900-0b94-11ea-9100-23daa035beea.png)

but it still works the same, so it should be fine 👍